pkg/awsManager/s3Manager.go

```diff
@@ -0,0 +1,59 @@
+package awsManager
+
+import (
+	"fmt"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/s3"
+	clientpkg "github.com/openshift/aws-account-shredder/pkg/aws"
+	"os"
+)
+
+// this does not delete the S3 instances , this only creates an []* string for the resources that have to deleted
+func ListS3InstancesForDeletion(assumedRoleClient clientpkg.Client) []*string {
+
+	var s3InstancesToBeDeleted []*string
+	s3bucketDescription, err := assumedRoleClient.ListBuckets(&s3.ListBucketsInput{})
+	if err != nil {
+		fmt.Println("ERROR:", err)
+		os.Exit(1)
+	}
+	for _, iterator := range s3bucketDescription.Buckets {
```

Comment:
`for _, bucket`

Reply:
This is still `iterator`

Reply:
resolved , i might have overlooked this.